src/assets/locales/hi.js

```diff
@@ -419,11 +419,8 @@ export default {
           "एकाधिक सही उत्तरों वाला बहुविकल्पीय प्रश्न जोड़ने के लिए यहां क्लिक करें",
       },
       item_editor: {
-        survey_mode: {
-          enabled: "सर्वेक्षण मोड में प्रश्नों के कोई सही उत्तर नहीं होते हैं",
-          disabled:
-            "प्लायो के प्रकाशित होने के बाद आप किसी सवाल का मोड नहीं बदल सकते",
-        },
+        survey_mode:
+          "सर्वेक्षण मोड में प्रश्नों के कोई सही उत्तर नहीं होते हैं",
```

Comment:
I think aman said somewhere to write सर्वे here right? I'm not sure if that part was resolved or not

Reply:
Yes, I thought only for the survey mode text, will do this.